simplyblock_cli/cli-reference.yaml

```diff
@@ -803,6 +803,11 @@ commands:
                 min: 0
                 max: 128
             default: 0
+          - name: "--fabric"
+            help: "fabric: tcp, rdma or both (specify: tcp, rdma)"
+            dest: fabric
+            type: str
```

Comment:
Here we can use key "choices" like this:
```
...
type: str
choices:
   - tcp
   - rdma
   - tcp,rdma
default: tcp
...
```

Reply:
Good one. Looks like we may eventually want an oneOrMore rule, but I think for now that's the only one. The choices is a good suggestion though.